Containers/clamav/start.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+
+if ! freshclam --foreground --stdout; then
+  exit 1
+fi
+
+sed "s|aio-placeholder|$MAX_SIZE|" /etc/clamav/clamd.conf > /tmp/clamd.conf
+
+freshclam --foreground --stdout --daemon & 
+clamd --foreground --config-file=/tmp/clamd.conf
```

Comment:
Shouldnt we better use supervisord for that? Then it will make sure to automatically restart both services if they should be stopped :)

Reply:
Thats how it is done in the start script of the official clamav image, but yes, feel free to edit this PR

Reply:
done